src/index.ejs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="{{.Locale}}">
```

Comment:
@m4dz it's a problem with mobile version I'm not sure it's a good idea to have the same index.ejs ;)